.yarnrc.yml

```diff
@@ -1,10 +1,9 @@
 nodeLinker: node-modules
 
-npmAuthToken: "${NPM_TOKEN}"
```

Comment:
I believe that [previously](https://github.com/aldrin-labs/ramm-sui-sdk/pull/6#discussion_r1469586967), I wrongly interpreted an authentication error in publishing `ramm-sui-sdk` to be caused by this field being nested inside `npmScopes`.

I corrected it in this comment and `yarn` continued to work as intended, so the cause was elsewhere.